validation/src/main/java/com/smartbank/validation/Config/KafkaTopicConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.smartbank.validation.Config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopicConfig {
+
+    @Value("${account.topic}")
+    private String accountTopic;
+
+    @Value("${transaction_request.topic}")
+    private String transaction_requestTopic;
```

Comment:
Naming convention is not good -> https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
Same for the one below

Reply:
done